src/peergos/server/mutable/MutableEventPropagator.java

```diff
@@ -0,0 +1,41 @@
+package peergos.server.mutable;
+
+import peergos.shared.crypto.hash.*;
+import peergos.shared.mutable.*;
+
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.function.*;
+
```

Comment:
docstring please.

Reply:
Why not  use ``java.util.Observer`` and ``java.util.Observable`` for this?

Reply:
Their signature doesn't match what we need.